src/wasm/wasm-binary.cpp

```diff
@@ -2056,7 +2056,13 @@ void WasmBinaryBuilder::readImports() {
         Name name(std::string("fimport$") + std::to_string(functionCounter++));
         auto index = getU32LEB();
         functionTypes.push_back(getTypeByIndex(index));
-        auto curr = builder.makeFunction(name, getTypeByIndex(index), {});
+        auto type = getTypeByIndex(index);
+        if (!type.isSignature()) {
+          throwError(std::string("Imported function ") + module.str + '.' +
+                     base.str +
```

Comment:
Can `module.str` and `base.str` be possibly null?

Reply:
In general it can be, but not for an imported function, which is the case here.